astropy/stats/src/compute_bounds.c

```diff
@@ -1,41 +1,58 @@
 #include "wirth_select.h"
 #include <math.h>
+#include <stdlib.h>
 
 void compute_sigma_clipped_bounds(double buffer[], int count, int use_median,
-                                  int maxiters, double sigma_lower,
+                                  int use_mad_std, int maxiters, double sigma_lower,
                                   double sigma_upper, double *lower_bound,
-                                  double *upper_bound) {
+                                  double *upper_bound, double buffer2[]) {
```

Comment:
I wonder if for clarity it would be better to call the first `buffer` something else, like `data[]` or `data_buffer[]`, and this one just `buffer` or `mad_buffer`

Reply:
Done